Sources/LiveViewNative/Coordinators/LiveViewCoordinator.swift

```diff
@@ -32,7 +32,7 @@ public class LiveViewCoordinator<R: RootRegistry>: ObservableObject {
         internalState
     }
     
-    @_spi(LiveForm) public let session: LiveSessionCoordinator<R>
+    @_spi(LiveForm) public weak var session: LiveSessionCoordinator<R>!
```

Comment:
This should be `private(set)`
```suggestion
    @_spi(LiveForm) public private(set) weak var session: LiveSessionCoordinator<R>!
```